utils/rss.ts

```diff
@@ -0,0 +1,26 @@
+import { v5 } from "uuid";
+
+// these constants are also located in catalog API - services/rss/library/rssUtils.js
+const podcastNamespace = "ead4c236-bf58-58c6-a2c6-a6b28d128cb6";
+const BASE_URL = "wavlake.com/feed";
+
+const getFeedUrl = (
+  contentType: "podcast" | "album" | "artist",
+  id: string,
+) => {
+  switch (contentType) {
+    case "podcast":
+      return `${BASE_URL}/show/${id}`;
+    case "album":
+      return `${BASE_URL}/music/${id}`;
```

Comment:
Open question: this is the newer path for albums that never was widely adopted. Should we use the old version instead? We'll have a redirect set up to reconcile these soon.

Reply:
Good catch, yeah I suppose let's use the old one

Reply:
Double-checking this in the rss service, it looks like we're using this new url to generate the feed guids there so this might be consistent.